common/helpers/helpers.py

```diff
@@ -24,3 +24,56 @@ def str_to_bool(s: Optional[str]) -> bool:
     Returns true if and only if s is defined and some variant capitalization of 'yes' or 'true'.
     """
     return s is not None and s.strip().upper() in ['YES', 'TRUE']
+
+
+  
+def dms2deg(dms):
+
+    if dms is None:
+        return None
+
+    d = float(dms[0])
+    m = float(dms[1])
+    s = float(dms[2])
+    sign = dms[3]
+    dd = d + m / 60. + s / 3600.
+
+    if sign == '-':
+        dd *= -1.
+
+    return dd
```

Comment:
I guess we should probably have typing information here?
`dms: Optional[List[float]]`
with a return type of `float`.